File: 1__C_Program/4__circle_circumference.c
#include <stdio.h>
#define PI 3.14159

int main () {
    // Declare variables and input diameter
    float diameter, circumference;
    printf("Enter diameter of a circle: "); scanf("%f", &diameter);

    // Calculate the circumference
    circumference = PI * diameter;
    printf("The circumference of a circle is: %.2f\n", circumference);
    return 0;
}

File: 5__JavaScript_Program/4__circle_circumference.js
const prompt = require('./node_modules/prompt-sync')({ sigint: true });

// Input diameter and print the circumference
const PI = 3.14159
const diameter = Number(prompt("Enter diameter of a circle: "));

const circumference = PI * diameter
console.log(`The circumference of a circle is: ${circumference}`);